ansible/playbooks/roles/kubernetes_node/tasks/node-join.yml

```diff
@@ -46,14 +46,6 @@
 
                 - <<: *soft-join
 
-              rescue:
-                - name: Join to cluster with ignores
-                  command: |-
-                    kubeadm join \
-                      --config /etc/kubeadm/kubeadm-join-node.yml \
-                      --ignore-preflight-errors all
-                  register: kubeadm_join_result
-
```

Comment:
In `always` block we have:

```
- name: Display kubeadm join stderr if any
  debug:
    msg: |
      Joined with warnings
      {{ kubeadm_join_result.stderr_lines }}
  when: kubeadm_join_result is failed
```

since the `rescue` block was removed, we may need to remove/adjust this `always` block?

Reply:
Rescue block is still there:

```
- block:
            - <<: &soft-join
                name: Join to Kubernetes cluster
                command: |-
                  kubeadm join \
                    --config /etc/kubeadm/kubeadm-join-node.yml
                register: kubeadm_join_result

          rescue:
            - name: Attempt to join with kubeadm reset
              when: kubeadm_join_result.stderr is search('/etc/kubernetes/kubelet.conf already exists')
              block:
                - name: Reset node
                  command: kubeadm reset --force

                - <<: *soft-join
```

Reply:
I meant the nested one (with `--ignore-preflight-errors all`) :)

Update: We decided to remove `always` block and keep only setting `node_already_joined` fact but outside the block.